### ISSUE TITLE
Mca mod compatability made textures.

### ISSUE THREAD

**Issue by SpGilbert** [PersonTheCat/ore_stone_variants#74]
Could you add mod compatibility for mca / Minecraft comes alive.
https://www.curseforge.com/minecraft/mc-mods/minecraft-comes-alive-mca
I spent some time making the dense ore textures and adding the base ore
![dense_mca_rose_gold_ore_overlay](https://user-images.githubusercontent.com/42950385/64131951-1efc0000-cd9a-11e9-9ce1-8f9887fc3b0a.png)
![mca_rose_gold_ore_overlay](https://user-images.githubusercontent.com/42950385/64131956-26230e00-cd9a-11e9-8a9a-b0d1599e9201.png)

**Comment by PersonTheCat** [comment]
Good idea. I might do this, but I really want to finish the 1.14 update first, which has proved to be super difficult. I'll hopefully be done with that before too long, though.

**Comment by SpGilbert** [comment]
ok

**Comment by SpGilbert** [comment]
how hard is it to add mod intigration for ore stone varients
